src/azul/chalice.py

```diff
@@ -65,6 +66,30 @@ def __init__(self, app_name, unit_test=False, spec=None):
         else:
             self._specs: Optional[MutableJSON] = None
         super().__init__(app_name, debug=config.debug > 0, configure_logs=False)
+        # The last handler is called first. We want logging to surround all
+        # other handler calls.
+        self.register_middleware(self.logging_middleware, 'http')
```

Comment:
IIRC the SQS -> Lambda interactions that the bundle notifications have use `sqs` as the event type rather than `http`, should we extend this logging middleware to have  `event_type=all` in order to record any type of event trigger that the lambdas? Not sure if we can do something like ('http', 'sqs') here.

Reply:
Glancing at the indexer lambdas, it seems like we already log when a notification comes in. Also middleware here would have to be changed since the SQS lambdas don't return a `response`.

If you think the SQS lambdas have insufficient logging, I think that would make sense as a separate issue.

Reply:
@amarjandu, I'm confused about the eyes emoji, lol.

Reply:
Me too.